.changeset/long-ears-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-yoga/plugin-persisted-queries': major
+---
+
+New Persisted Queries Plugin
```

Comment:
Let’s add an example part of changeset and also let’s add docs so when we cut a release we don’t have to spend more time documenting we can just do as we go.

Reply:
Done! Let me know what you think.